frontend/src/component/environments/EnvironmentTable/EnvironmentTable.tsx

```diff
@@ -61,26 +67,32 @@ export const EnvironmentTable = () => {
     );
 
     const columnsWithActions = useMemo(() => {
+        const baseColumns = [...COLUMNS];
+        if (globalChangeRequestConfigEnabled && isEnterprise()) {
+            baseColumns.splice(2, 0, {
+                Header: 'Change request',
+                accessor: (row: IEnvironment) =>
+                    Number.isInteger(row.requiredApprovals) ? 'yes' : 'no',
```

Comment:
requiredApprovals is either null or 1,2,3,4. 0 is not allowed in the openapi schema